features/search.feature

```diff
@@ -11,41 +11,3 @@ Feature: Searching
     Given I am on the home page
     When I search for something relevant
     Then I should see search results
-
-  Scenario: A relevant message is displayed when no results are found
-    Given I am on the home page
-    When I search for something irrelevant
-    Then I should see no search results
-    And I should prompted to try another search term
```

Comment:
what is the expectation of this scenario now?

Reply:
There is no spelling suggestion out of the box on Algolia. It is a feature that has been deleted.

Reply:
ah, so this was based on spelling suggestions not just a generic 'no results found' message then?